components/Statistics/Iceberg/RecentlyPlayedTracks.vue

```diff
@@ -0,0 +1,43 @@
+<script lang="ts" setup>
```

Comment:
RecentlyPlayedTracks würde ich nicht in diesem PR mitnehmen.
1. Es ist in Fällen von nicht so aktiven Nutzern identisch zu TopTracks.
2. Es passt nicht zu der Timeframe selection oben. Das führt einige Probleme ein, die geklärt werden müssen, bevor wir das einbinden können.

Wenn die Komponente dir am herzen liegt, können wir das in einem anderen PR nachziehen